src/pages/billing/Project.js

```diff
@@ -21,17 +21,21 @@ import { billingRoles } from 'src/pages/billing/List'
 
 
 const workspaceLastModifiedWidth = 150
+const workspaceExpandIconSize = 20
 
 const WorkspaceCardHeaders = Utils.memoWithName('WorkspaceCardHeaders', ({ sort, onSort }) => {
   return div({ style: { display: 'flex', justifyContent: 'space-between', marginTop: '1.5rem', padding: '0 1rem', marginBottom: '0.5rem' } }, [
-    div({ 'aria-sort': ariaSort(sort, 'name'), style: { flex: 1 } }, [
+    div({ 'aria-sort': ariaSort(sort, 'name'), style: { flex: 1, paddingLeft: '1rem' } }, [
       h(HeaderRenderer, { sort, onSort, name: 'name' })
     ]),
     div({ 'aria-sort': ariaSort(sort, 'createdBy'), style: { flex: 1 } }, [
       h(HeaderRenderer, { sort, onSort, name: 'createdBy' })
     ]),
     div({ 'aria-sort': ariaSort(sort, 'lastModified'), style: { flex: `0 0 ${workspaceLastModifiedWidth}px` } }, [
       h(HeaderRenderer, { sort, onSort, name: 'lastModified' })
+    ]),
+    div({ role: 'columnheader', style: { flex: `0 0 ${workspaceExpandIconSize}px` } }, [
```

Comment:
Thank you for thinking to put the column header in! But actually, since this is no longer a table, this will probably be flagged by aXe as being a columnheader floating around outside a table. I'd say for now you don't need this, unless you change it back to being a table at a later point.

Reply:
Incidentally, this is an issue the `axe-core` linter should be able to catch. It very easily catches too-much-ARIA, it's the not-enough-ARIA case that it sometimes struggles with. :)